File: update_script.js
const fs = require('fs');
let text = fs.readFileSync('script.js', 'utf8');

text = text.replace(/< span style = "font-size:12px;color:#999;" > /g, '<span style="font-size:12px;color:#999;">');
text = text.replace(/ <\/span > /g, '</span>');
text = text.replace(/\$\{ t\.desc \}/g, '${t.desc}');
text = text.replace(/\$\{ dt\.getMonth\(\) \+ 1 \}/g, '${dt.getMonth() + 1}');
text = text.replace(/ \`分类明细：\$\{ catName \} \`/g, '`分類明細：${catName}`');

fs.writeFileSync('script.js', text, 'utf8');

File: update_script.py
import codecs
with codecs.open('script.js','r', 'utf-8') as f:
    text = f.read()

text = text.replace('< span style = "font-size:12px;color:#999;" > ', '<span style="font-size:12px;color:#999;">')
text = text.replace(' </span > ', '</span>')
text = text.replace('${ t.desc }', '${t.desc}')
text = text.replace('${ dt.getMonth() + 1 }', '${dt.getMonth() + 1}')

with codecs.open('script.js','w', 'utf-8') as f:
    f.write(text)
